src/hugo/content/en-us/implementations/aws/command/command2/_index.md

```diff
@@ -0,0 +1,721 @@
+        update_accepted_subscribed_future, _ = shadow_client.subscribe_to_update_shadow_accepted(
+            request=iotshadow.UpdateShadowSubscriptionRequest(thing_name=thing_name),
+            qos=mqtt.QoS.AT_LEAST_ONCE,
+            callback=on_update_shadow_accepted)
+
+        update_rejected_subscribed_future, _ = shadow_client.subscribe_to_update_shadow_rejected(
+            request=iotshadow.UpdateShadowSubscriptionRequest(thing_name=thing_name),
+            qos=mqtt.QoS.AT_LEAST_ONCE,
+            callback=on_update_shadow_rejected)
+
+        # Wait for subscriptions to succeed
+        update_accepted_subscribed_future.result()
+        update_rejected_subscribed_future.result()
+
+        print("Subscribing to Get responses...")
+        get_accepted_subscribed_future, _ = shadow_client.subscribe_to_get_shadow_accepted(
+            request=iotshadow.GetShadowSubscriptionRequest(thing_name=thing_name),
+            qos=mqtt.QoS.AT_LEAST_ONCE,
+            callback=on_get_shadow_accepted)
+
+        get_rejected_subscribed_future, _ = shadow_client.subscribe_to_get_shadow_rejected(
+            request=iotshadow.GetShadowSubscriptionRequest(thing_name=thing_name),
+            qos=mqtt.QoS.AT_LEAST_ONCE,
+            callback=on_get_shadow_rejected)
+
+        # Wait for subscriptions to succeed
+        get_accepted_subscribed_future.result()
+        get_rejected_subscribed_future.result()
+
+        print("Subscribing to Delta events...")
+        delta_subscribed_future, _ = shadow_client.subscribe_to_shadow_delta_updated_events(
+            request=iotshadow.ShadowDeltaUpdatedSubscriptionRequest(thing_name=thing_name),
+            qos=mqtt.QoS.AT_LEAST_ONCE,
+            callback=on_shadow_delta_updated)
+
+        # Wait for subscription to succeed
+        delta_subscribed_future.result()
+
+
+        # Issue request for shadow's current state.
+        # The response will be received by the on_get_accepted() callback
+        print("Requesting current shadow state...")
+
+        # use a unique token so we can correlate this "request" message to
+        # any "response" messages received on the /accepted and /rejected topics
+        token = client_token
+
+        publish_get_future = shadow_client.publish_get_shadow(
+            request=iotshadow.GetShadowRequest(thing_name=thing_name, client_token=token),
+            qos=mqtt.QoS.AT_LEAST_ONCE)
+     
+        # Ensure that publish succeeds
+        publish_get_future.result()
+
+    except Exception as e:
+        exit(e)
+
+    # Wait timer to receive delta events when device remain connected after initial connect/reconnect   
+    try:
+        while True:
+            time.sleep(1)
+    except KeyboardInterrupt:
+        # Disconnect
+        print("Disconnecting...")
+        disconnect_future = mqtt_connection.disconnect()
+        disconnect_future.result()
+        print("Disconnected!")
+
+```
+- Update shadow request document 
```

Comment:
How and where is this updated, locally from the client code above or via the CLI/console? If the latter, put in instructions on how to achieve.